infra/index.ts

```diff
@@ -1312,11 +1290,15 @@ const rabbitmqDeployment = new k8s.apps.v1.Deployment("rabbitmq-deployment", {
 
 const uiImage = new awsx.ecr.Image("ui-image", {
     repositoryUrl: repository.url,
-    dockerfile: "../images/java17/Dockerfile",
-    path: "../src/ui",
+    dockerfile: "/Users/slowe/Work/Code/Repos/zephyr-app/images/java17/Dockerfile",
+    path: "/Users/slowe/Work/Code/Repos/zephyr-app/src/ui",
```

Comment:
Should these be something else?

Reply:
Yes, I'll need to fix this. Thanks for catching it!

Reply:
Fixed in 624ec89